clients/cli/src/clap_app.rs

```diff
@@ -2688,4 +2703,46 @@ pub fn app<'a>(
                 .arg(multisig_signer_arg())
                 .nonce_args(true)
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateUiMultiplier.into())
+                .about("Update UI multiplier")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(|s| is_valid_pubkey(s))
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The token address with scaled UI amount multiplier"),
+                )
+                .arg(
+                    Arg::with_name("ui-multiplier")
+                        .value_name("MULTIPLIER")
```

Comment:
nit: we typically use underscores in the arg name, but maybe we can simplify it to `multiplier` while we're at it?
```suggestion
                    Arg::with_name("multiplier")
                        .value_name("MULTIPLIER")
```